servlet/spring-boot/java/oauth2/azure-active-directory-b2c/resource-server-1-simple-sample/src/main/java/com/spring/sample/b2c/controller/SampleController.java

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.spring.sample.b2c.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+public class SampleController {
+
+    @ResponseBody
+    @GetMapping(value = { "/hello" })
+    public String hello() {
+        return "Hello, this is aad-b2c-resource-server-simple-sample.";
```

Comment:
`this is a resource-server protected by Azure Active Directory B2C. ` Then we can reuse this if we adding other resource server projects.